connexion/decorators/validation.py

```diff
@@ -105,16 +112,30 @@ def __call__(self, function):
         def wrapper(request):
             if all_json(self.consumes):
                 data = request.json
-
                 if data is None and len(request.body) > 0 and not self.is_null_value_valid:
                     # the body has contents that were not parsed as JSON
                     return problem(415,
-                                   "Unsupported Media Type",
```

Comment:
Being working on major changes, I'd avoid typographical changes as they increase the review work.

Reply:
good catch, thanks!